engines/agos/detection.cpp

```diff
@@ -51,11 +51,11 @@ static const DebugChannelDef debugFlagList[] = {
 
 static const PlainGameDescriptor agosGames[] = {
 	{"pn", "Personal Nightmare"},
-	{"elvira1", "Elvira - Mistress of the Dark"},
-	{"elvira2", "Elvira II - The Jaws of Cerberus"},
+	{"elvira1", "Elvira: Mistress of the Dark"},
+	{"elvira2", "Elvira II: The Jaws of Cerberus"},
 	{"waxworks", "Waxworks"},
-	{"simon1", "Simon the Sorcerer 1"},
-	{"simon2", "Simon the Sorcerer 2"},
+	{"simon1", "Simon the Sorcerer"},
+	{"simon2", "Simon the Sorcerer 2: The Lion, the Wizard and the Wardrobe"},
```

Comment:
This one should be "Simon the Sorcerer II: The Lion, the Wizard and the Wardrobe"

Reply:
Good catch